packages/asl-pages/pages/role/named-person-mvp/select-role/index.js

```diff
@@ -79,11 +79,6 @@ module.exports = (settings) => {
 
         next();
       },
-      getValues: (req, res, next) => {
-        req.form.values.rcvsNumber =
-          req.form.values.rcvsNumber || req.profile.rcvsNumber;
-        next();
-      },
```

Comment:
This is [existing behaviour](https://github.com/UKHomeOffice/aspel-workspace/blob/main/packages/asl-pages/pages/role/apply/index.js#L91) and is copying the applicant's existing number if they have one into subsequent role applications, which feels useful.

Reply:
```js
      getValues: (req, res, next) => {
        req.form.values.rcvsNumber =
          req.form.values.rcvsNumber ?? req.profile.rcvsNumber;
        next();
      },
```

Might be better? If the user explictly deletes it then it will be `''` rather than `null`/`undefined`. 

There isn't really a use case for wanting to delete it though, and we haven't had issues with it working like this on the existing journey. Someone being nominated for an NVS role needs to have an RCVS number and it's unlikely to change for that user. If entered incorrectly the correction won't be falsey so won't get reset.